code/modules/clothing/head/misc_special.dm

```diff
@@ -234,3 +234,13 @@
 		if(~initial(user.species.flags) & HYPOTHERMIA_IMMUNE)
 			user.species.flags &= ~HYPOTHERMIA_IMMUNE
 		user.faction = initial(user.faction)
+
+/obj/item/clothing/head/beret/sec/ocelot
+	name = "Ocelot's beret"
+	desc = "Ocelot's signature red beret"
+
+/obj/item/clothing/head/beret/sec/ocelot/OnMobLife(var/mob/living/carbon/human/wearer)
+	var/mob/living/carbon/human/W = wearer
```

Comment:
there you go, I just copied from how the happiest mask does it so can you explain why that mask does it that way instead?